Description: Do not skip tests.
 If Build-Depends are satisfied, importing ElementTree should not fail. If it
 does, it's better to fail than to skip the test silently.
Author: Jakub Wilk <jwilk@debian.org>
Forwarded: not-needed
Last-Update: 2012-03-02

--- a/test/test_writers/test_odt.py
+++ b/test/test_writers/test_odt.py
@@ -74,6 +74,7 @@
                     from elementtree import ElementTree as etree
                     WhichElementTree = 'elementtree'
                 except ImportError, e:
+                    raise
                     s1 = '\nSkipped test of odf_odt writer.  ' \
                          'In order to test odf_odt writer ' \
                          'must install either a version of Python containing ' \
